src/Illuminate/Database/Query/Builder.php

```diff
@@ -63,7 +63,17 @@ class Builder implements BuilderContract
     /**
      * The current query value bindings.
      *
-     * @var array
+     * @var array{
+     *     select: list<mixed>,
+     *     from: list<mixed>,
+     *     join: list<mixed>,
+     *     where: list<mixed>,
+     *     groupBy: list<mixed>,
+     *     having: list<mixed>,
+     *     order: list<mixed>,
+     *     union: list<mixed>,
+     *     unionOrder: list<mixed>,
+     * }
```

Comment:
Better than nothing but are we really unable to narrow `mixed` further down?

Reply:
Well, what do you suggest then?